src/eppo_value.ts

```diff
@@ -55,15 +57,15 @@ export class EppoValue {
 
   toString(): string {
     switch (this.valueType) {
-      case ValueType.NullType:
+      case EppoValueType.NullType:
         return 'null';
-      case ValueType.BoolType:
+      case EppoValueType.BoolType:
         return this.boolValue ? 'true' : 'false';
-      case ValueType.NumericType:
+      case EppoValueType.NumericType:
         return this.numericValue ? this.numericValue.toString() : '0';
```

Comment:
This is the time to fix everything -- but I'm not quite following the change you have in mind; could you expand?

Reply:
Like let say a new numeric eppo value with a null value is initialized. This would output `0` for its value which really isn't true. I'm saying any EppoValue initialized with null/undefined value should be an `EppoValueType.NullType`. Anything that isn't, should have a non-null value. Also, any type of check would fail against that, returning the default value.

Reply:
Okay yes I see your point now... but we should simply not allow creating a numeric value with a null value to begin with it seems